model/BUILD.bazel

```diff
@@ -223,6 +223,18 @@ java_lite_proto_library(
     deps = [":exploration_proto"],
 )
 
+format_import_proto_library(
+    name = "platform_parameter",
+    src = "src/main/proto/platform_parameter.proto",
+    visibility = ["//:oppia_api_visibility"],
```

Comment:
You can omit this visibility since it will be used by java_lite_proto_library in this same package.

Reply:
Done